airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingConsumerFactory.java

```diff
@@ -170,11 +172,18 @@ private static OnCloseFunction onCloseFunction(final JdbcDatabase database,
           LOGGER.info("Finalizing stream {}. schema {}, tmp table {}, final table {}", writeConfig.getStreamName(), schemaName, srcTableName,
               dstTableName);
 
-          final String stageName = namingResolver.getStageName(schemaName, dstTableName);
-          sqlOperations.copyIntoTmpTableFromStage(database, stageName, srcTableName, schemaName);
-          LOGGER.info("Uploading data from stage:  stream {}. schema {}, tmp table {}, stage {}", writeConfig.getStreamName(), schemaName,
-              srcTableName,
-              stageName);
+          final String path = namingResolver.getStagingPath(schemaName, dstTableName, currentSyncPath);
+          LOGGER.info("Uploading data from stage:  stream {}. schema {}, tmp table {}, stage path {}", writeConfig.getStreamName(), schemaName,
+                  srcTableName,
+                  path);
+          try {
+            sqlOperations.copyIntoTmpTableFromStage(database, path, srcTableName, schemaName);
+          }catch (Exception e){
```

Comment:
Looks like one space missed here before the catch, please reformat.

Reply:
fixed